### PR TITLE
feat(home-page): create 1990 designs home page

### PR DESCRIPTION
**Pull request by Bradogilo**
#### What does this PR do?

It creates a non-responsive home page with all its scenarios for 1990 designs.

#### Tasks to be completed
- add navigation bar
- add hero section
- add call-to-action buttons
- add footer
- add social media links
- add newsletter section
- add footer navigation items

#### Relevant Netlify Link

- https://1990designs.netlify.com/

#### Relevant Screenshots

Screenshot 2020-03-16 at 02 30 41
Screenshot 2020-03-16 at 02 30 54

### CODE REVIEW

**Comment by Bradogilo** [comment]
[Homepage](https://trello.com/c/U0Z3vWZL/3-homepage)

File: index.html [hunk 1]
```diff
@@ -2,11 +2,122 @@
...
+                    <p class="digital">digital</p>
+                </section>
+            </section>
+            <!-- end of what I do -->
+
+            <!-- work -->
+            <section class="portfolio">
+                <a href="/portfolio.html"><h4 class="portfolio-title">portfolio</h4></a>
+                <section class="portfolio-items">
+                    <p class="portfolio-strategy">all categories</p>
```
Comment (DerKip) [inline index.html]: put these as links in a ul
```suggestion
<ul>
<li>< a href="#">all categories</a></li>
<li></li>
</ul>
```

Comment (Bradogilo) [reply index.html]: Yeah, I think so too. Let me change that. Thanks for the feedback.

Comment (Bradogilo) [reply index.html]: This has been updated @DerKip

**Pull request merged**